templates/editConn.js

```diff
@@ -99,6 +99,12 @@ class Actions {
       connOrg: ''
     }
 
+    // trim trailing slash on connHost input
+    let host = result.connHost
+    if (host[host.length-1] === '/') {
```

Comment:
Should we check the length first? Or is JS going to shield us from any issues if connHost is length 0.

Reply:
Normally, yes we would check the length, but the `getData` function is only called in two places, both of which are preceded by a call to `validate`, which will show the user en error if there isn't a properly formatted url in the input.

https://github.com/influxdata/vsflux/blob/d9f6859cbe248efa73f1d53846fb52435ee0d7b6/templates/editConn.js#L14-L32

Reply:
Ah, okay, thanks. Had a quick look but didn't see that.